src/ui/views/settings/providers/taskwarrior.rs

```diff
@@ -8,6 +8,10 @@ use druid_widget_nursery::prism::Prism;
 fn taskwarrior_settings() -> impl ProviderSettingsBuilder<TaskwarriorConfig> {
     SettingsBuilder::new("Taskwarrior")
         .add_field(ProviderSettingsRow::new("Query", TaskwarriorConfig::query).multiline())
+        .add_flag_field(ProviderSettingsFlagRow::new(
```

Comment:
Ha, ich dachte ich unterstütze noch keine boolean fields in den settings. Deshalb hatte ich das nich angemerkt :D